### DISCUSSION TITLE
Complete the expression

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1009] [Galaxy Trivia] [answered]
> It is better to be hated for what you are than ...

**Bonus:** Name the author.

**Comment by peterbaccaro** [comment]
The phrase **It is better to be hated for what you are than to be loved for what you are not** is commonly attributed to French author and philosopher **André Gide**.

**Comment by thuongtruong1009** [comment, marked as answer]
It is better to be hated for what you are than loved for what you are not.

The author of this quote is André Gide, a French author and winner of the Nobel Prize in Literature in 1947.
